upgrade/scripts/upgrade/util/update-customizations.sh

```diff
@@ -73,7 +73,7 @@ trap 'rm -f $c' EXIT
 
 cp "$customizations" "$c"
 
-# argo/cray-nls
+# argo/cray-s
```

Comment:
What is the purpose of this change?

Reply:
This happened by mistake so marking it as resolved.